packages/tasks/src/model-libraries.ts

```diff
@@ -680,6 +680,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"safetensors" OR path_extension:"pt"`,
 	},
+	"mochi-1-preview": {
+		prettyLabel: "Mochi Preview",
+		repoName: "mochi-1-preview",
+		repoUrl: "https://github.com/genmoai/models",
+		filter: false,
+		countDownloads: `path_extension:"safetensors" OR path_extension:"pt"`,
```

Comment:
In [the repo](https://huggingface.co/genmo/mochi-1-preview/tree/main), I see two safetensors files that presumably go together, and no `pt` files. In order to prevent double-counting, would it make sense to count `dit-config.yaml` instead?

Reply:
dit-config.yaml should be removed, ideally we should just count one of the safetensor files.